plugins/core/Standard/qCompass/ccCompass.cpp

```diff
@@ -4445,6 +4451,21 @@ int ccCompass::writeObjectXML(ccHObject* object, QXmlStreamWriter* out)
 		}
 	}
 
+	//write facet data
+	if (object->isA(CC_TYPES::FACET))
+	{
+		//write orientation
+		ccFacet* f = static_cast<ccFacet*>(object);
+		out->writeTextElement("Nx", QString::asprintf("%f", f->getNormal().x));
+		out->writeTextElement("Ny", QString::asprintf("%f", f->getNormal().y));
+		out->writeTextElement("Nz", QString::asprintf("%f", f->getNormal().z));
+		out->writeTextElement("Cx", QString::asprintf("%f", f->getCenter().x));
+		out->writeTextElement("Cy", QString::asprintf("%f", f->getCenter().x));
```

Comment:
This doesn't seem right, shouldn't this be `f->getCenter().y`? Similarly in the line below

Reply:
Good catch Bedzior - a sloppy bug on my part. Thanks!